cypher-shell/src/test/java/org/neo4j/shell/commands/SetTest.java

```diff
@@ -0,0 +1,67 @@
+package org.neo4j.shell.commands;
+
+
+import org.junit.Before;
+import org.junit.Test;
+import org.neo4j.shell.Command;
+import org.neo4j.shell.TestShell;
+import org.neo4j.shell.exception.CommandException;
+
+import static junit.framework.TestCase.assertTrue;
+import static junit.framework.TestCase.fail;
+import static org.junit.Assert.assertEquals;
+
+public class SetTest {
+
+    private TestShell shell = new TestShell();
+    private Command cmd;
+
+    @Before
+    public void setup() {
+        this.cmd = new Set(shell);
```

Comment:
Once merged, I will try rewriting this with Mockito to mimic how the other commands are being tested.